gallery/plot_repurposing_annotations.py

```diff
@@ -0,0 +1,75 @@
+"""
+=======================
+Repurposing annotations
+=======================
+
+The following example illustrates the operations available in :ref:`the torchvision.ops module <ops>` for repurposing
+object localization annotations for different tasks (e.g. transforming masks used by instance and panoptic
+segmentation methods into bounding boxes used by object detection methods).
+"""
+import os.path
+
+import PIL.Image
+import matplotlib.patches
+import matplotlib.pyplot
+import numpy
+import torch
+import torchvision.ops.boxes
+
+ASSETS_DIRECTORY = "../test/assets"
```

Comment:
Actually there is gallery assets folder.
We shouldn't use the tests folder I think

https://github.com/pytorch/vision/tree/main/gallery/assets

Reply:
I didn't want to duplicate assets.

Reply:
Correct. But it would be nice to seperate tests from gallery. Let's hear from others too!